src/client/pages/CountryHome/Repository/Panel/Panel.tsx

```diff
@@ -1,33 +1,38 @@
 import './Panel.scss'
 import React from 'react'
 
+import { Objects } from 'utils/objects'
+
+import { useRepositoryItem } from 'client/store/ui/repository/hooks'
 import SlidingPanel from 'client/components/SlidingPanel'
 import Actions from 'client/pages/CountryHome/Repository/Panel/Actions'
+import { useClosePanel } from 'client/pages/CountryHome/Repository/Panel/hooks/useClosePanel'
 import InputField from 'client/pages/CountryHome/Repository/Panel/InputField'
 import Separator from 'client/pages/CountryHome/Repository/Panel/Separator'
 
 import { useOnChange } from './hooks/useOnChange'
-import { useOnSaveFile } from './hooks/useOnSaveFile'
-
-interface Props {
-  openPanel: boolean
-  setOpenPanel: (open: boolean) => void
-}
 
-const Panel: React.FC<Props> = (props: Props) => {
-  const { openPanel, setOpenPanel } = props
+const Panel: React.FC = () => {
+  const repositoryItem = useRepositoryItem()
+  const openPanel = !Objects.isEmpty(repositoryItem)
 
-  const { file, onChange } = useOnChange()
-  const onSaveFile = useOnSaveFile(file, setOpenPanel)
+  const onChange = useOnChange()
+  const closePanel = useClosePanel()
 
   return (
-    <SlidingPanel openPanel={openPanel} setOpenPanel={setOpenPanel}>
+    <SlidingPanel openPanel={openPanel} setOpenPanel={closePanel}>
       <div className="repository-form__container">
-        <InputField onChange={onChange} label="editUser.name" name="name" type="text" />
-        <InputField onChange={onChange} label="common.link" name="link" type="text" />
+        <InputField value={repositoryItem?.name} onChange={onChange} label="editUser.name" name="name" type="text" />
+        <InputField value={repositoryItem?.link} onChange={onChange} label="common.link" name="link" type="text" />
         <Separator />
-        <InputField onChange={onChange} label="common.file" name="file" type="file" />
-        <Actions onSave={onSaveFile} setOpenPanel={setOpenPanel} />
+        <InputField
+          // todo
+          onChange={onChange}
+          label="common.file"
+          name="file"
+          type="file"
+        />
```

Comment:
TODO:
Find a good solution to keep the file somewhere else than redux store
Possible just keep in local state using `useState` as before